src/init.cpp

```diff
@@ -457,6 +458,14 @@ static void BlockNotifyCallback(const uint256& hashNewTip)
     boost::thread t(runCommand, strCmd); // thread runs free
 }
 
+static void TxNotifyCallback(const uint256& txid)
+{
+    std::string strCmd = GetArg("-txnotify", "");
+
+    boost::replace_all(strCmd, "%s", txid.GetHex());
+    boost::thread t(runCommand, strCmd); // thread runs free
```

Comment:
I think it is a bad idea to start a process for each accepted tx.
Why not supporting https://github.com/bitcoin/bitcoin/pull/6103 and go over sockets?